lib/repo_tools.pm

```diff
@@ -100,27 +108,40 @@ sub rmt_wizard {
     zypper_call 'in rmt-server';
     zypper_call 'in mariadb';
 
-    # check mysql status and config mysql for RMT
-    systemctl 'start mysql.service';
-    systemctl 'status mysql.service';
-    my $cmd = 'mysql -u root -p <<EOFF
-GRANT ALL PRIVILEGES ON \`rmt\`.* TO rmt@localhost IDENTIFIED BY \'rmt\';
-FLUSH PRIVILEGES;
-EOFF';
-    type_string "$cmd\n";
-    assert_screen('rmt-sqladmin-password', 40);
-    send_key 'ret';
-
-    # Modify rmt config file
-    assert_script_run "sed -i '/^[][ ]*username:[][ ]*\$/d' /etc/rmt.conf";
-    assert_script_run "sed -i '/^[][ ]*password/d' /etc/rmt.conf";
-    assert_script_run "sed -i '/adapter/i\ \\ \\ password: rmt' /etc/rmt.conf";
-    assert_script_run "sed -i '/scc/a\ \\ \\ password: $SCC_Password' /etc/rmt.conf";
-    assert_script_run "sed -i '/scc/a\ \\ \\ username: $SCC_User' /etc/rmt.conf";
-
-    # Start rmt server
-    systemctl 'start rmt';
-    systemctl 'status rmt';
+    type_string "yast2 rmt;echo yast2-rmt-wizard-\$? > /dev/$serialdev\n";
+    assert_screen 'yast2_rmt_registration';
+    send_key 'alt-u';
+    wait_still_screen;
+    type_string(get_required_var('SMT_ORG_NAME'));
+    send_key 'alt-p';
+    wait_still_screen;
+    type_string(get_required_var('SMT_ORG_PASSWORD'));
+    send_key 'alt-n';
+    assert_screen 'yast2_rmt_config_written_successfully';
+    send_key 'alt-o';
+    wait_still_screen;
+    assert_screen 'yast2_rmt_db_password';
```

Comment:
we should not need `wait_still_screen` in case you have `assert_screen`. Maybe in exceptional cases only.

Reply:
Yes, removed such lines.